### ISSUE TITLE
obj loss increases during the training

### ISSUE THREAD

**Issue by liamsun2019** [MohsenZand/ObjectBox#3]
Hi author,
Appreicate sharing your excellent work. My use case:
1. one class: person
2. format the label file with yolo style, e.g:
0 0.76328125 0.5339578454332553 0.0546875 0.1592505854800937
where the 5 elements are: clsid, centerx, centery, boxw, boxh, all of which are normalized against original image width/height
3. All the train/val images are filtered out from coco train2017/val2017 with category being "person"
4. Modify flag_sets.py, objectBox_COCO.yaml and coco.yaml, especially set nc to 1 for my case
5. The training shows that the obj loss increases while box loss decreases.

I am curious how this could happen. Any suggestions? Thanks.

**Comment by MohsenZand** [comment]
Thank you!
Check the ground-truths for generated training samples ('train_batchX.jpg') in your project folder to ensure that bounding boxes are only and accurately for 'persons'. 
In 'config/hyp.yaml', you should additionally tweak the related hyperparameters (such as obj and obj_pw).
